tests/nix-channel.sh

```diff
@@ -33,7 +33,10 @@ grep -q 'item.*attrPath="foo".*name="dependencies-top"' $TEST_ROOT/meta.xml
 
 # Do an install.
 nix-env -i dependencies-top
-[ -e $TEST_HOME/.nix-profile/foobar ]
+[ -e $TEST_HOME/.local/state/nix/profile/foobar ]
+
+clearProfiles
+rm -rf $TEST_HOME/.local/state/nix
```

Comment:
test is faling because of this and 
`grep -q 'item.*attrPath="foo".*name="dependencies-top"' $TEST_ROOT/meta.xml`

foo is added above but its cleared and only bar is added below

Reply:
removed that grep and pushed it in my branch

Reply:
nevermind its actually intentional 6f1b3d711e6174286ae8ecb83c5d8a348e9fa2c3

@balsoft why was this added?